src/voice/audio.rs

```diff
@@ -7,7 +7,7 @@ pub const SAMPLE_RATE: u32 = 48_000;
 
 /// A readable audio source.
 pub trait AudioSource: Send {
-    fn is_stereo(&mut self) -> bool;
```

Comment:
Irrelevant [to this pr] and (possibly?) an unnecessary breaking change. So revert this please.

Reply:
I mean it should just read a value, and the only use does that, but you're right it's unrelated

Reply:
![](https://i.imgur.com/uhC6A7I.png)

Reply:
that's why I did it